doc/developer-guide/deployment/https.rst

```diff
@@ -3,9 +3,10 @@
 HTTPS support
 =============
 
-Zotonic has built-in support for https and certificate handling.
+Zotonic has built-in support for HTTPS and TLS (previously SSL) certificate
+handling.
 
-All sites share a single ssl port using Server Name Indication for 
+All sites share a single TLS port using Server Name Indication for
```

Comment:
Big provider Squarespace (also) recently decided on maintaining the SSL nomenclature (instead of TLS): https://support.squarespace.com/hc/en-us/articles/205815898-Squarespace-and-SSL